LongWarOfTheChosen/Config/XComGameData_CharacterStats.ini

```diff
@@ -8867,62 +8867,174 @@ CharacterBaseStats[eStat_FlankingCritChance]=40
 [AdvPurifierM1 X2CharacterTemplate]
 CharacterBaseStats[eStat_ArmorMitigation]=1
 CharacterBaseStats[eStat_HP]=6
-+CharacterBaseStats[eStat_Offense]=160
+CharacterBaseStats[eStat_Offense]=65
+CharacterBaseStats[eStat_Mobility]=10
```

Comment:
Wasn't the plan for this to be 9, or is it 9 after the weights of the pistol and grenade are included?

Reply:
The latter